prisma/schema.prisma

```diff
@@ -118,36 +118,36 @@ model Area {
   questions            Question[]
   assessors_guide      String?
   stageId              Int
-  stage                Stage      @relation(fields: [stageId], references: [id])
+  stage                Stage       @relation(fields: [stageId], references: [id])
   artefacts            Artefact[]
-
-  @@unique([area_id, area_name, stageId])
+  areaScores           AreaScore[] // This is the opposite relation field
```

Comment:
Why adding all areaScores from all users and all projects to Area table? I think @severynenko's idea is to avoid this

Reply:
This relationship already exists. Prisma validation requires that the relationship is present on both ends of the models. @NingyueZhou Pleade double check that you're not just confusing Prisma's schema validation requirements with our model requirements.

https://www.prisma.io/docs/orm/prisma-schema/data-model/relations

I am not challenging the schema logic but simply fixing the validation errors so that we can actually be able to push the schema to the database...

Reply:
Ah now I understand, thank you for fixing this!